pom.xml

```diff
@@ -6,8 +6,7 @@
 	<groupId>life.qbic</groupId>
 	<artifactId>experimental-design-lib</artifactId>
 	<packaging>jar</packaging>
-	<version>0.20.3</version>
-
+	<version>0.21.0-SNAPSHOT</version>
```

Comment:
I'd also advise to remove the duplicate declaration of the `maven compiler plugin` in the plugin section in line 94 of the pom.xml